changelog.md

```diff
@@ -1,5 +1,23 @@
 # Changelog
 
+## 1.11
```

Comment:
just add all these to version 1.10 I am going to be making some changes likely before I publish.